opensearch-dashboard-prototyping/install_dashboards.sh

```diff
@@ -5,21 +5,28 @@ MAJOR='\033[0;34m'
 MINOR='\033[0;37m    '
 RESET='\033[0m' # No Color
 
-: ${OPEN_SEARCH:="localhost:9200"}
```

Comment:
You can do this if you want default values:

```
OPEN_SEARCH=${1:-"default value if not provided"}
OPEN_SEARCH_DASHBOARDS=${2:-"default value if not provided"}
```

Reply:
i kinda just prefer variables...   versus environment variables...   for explicitness...

Reply:
Those are local variables.